.github/workflows/cluster-rotate.yml

```diff
@@ -98,9 +98,12 @@ jobs:
         run: |
           if [ -f "${ARTIFACT_DIR}/kubeconfig" ]; then
             export KUBECONFIG="${ARTIFACT_DIR}/kubeconfig"
-            for n in $(kubectl get ns -o=custom-columns=kube-:.metadata.name | grep -v "kube-\|default"); do
-              kubectl delete ns $n
-            done
+            kubectl get ns -o=custom-columns=kube-:.metadata.name | grep -v \
```

Comment:
Hm, I would rather go with `--no-headers` so you don't have to create a header that you'll be ignoring with grep, just for clarity, the result is the same of course...

Reply:
I was not aware of the `--no-headers` I will make that change, thanks!